readme.md

```diff
@@ -283,15 +283,33 @@ curl -X POST -H "Content-Type: application/json" \
     http://localhost:3000/v2/encoder/obisObserver
 ```
 
+## Adapters
+
+Adapters are used to convert data from an external source to an internal format.
+Each adapter requires the HTTP header `ns-adapter` to be set to one of the available adapters.
+Available adapters: ChirpStack.
+
+#### ChirpStack
+
+1. Setup `protocol` tag in device profile as one of `analog`, `mtx`, 'obisObserver`
```

Comment:
should be `obisObserver` instead 'obisObserver`

Reply:
fixed